taxonomies/categories.txt

```diff
@@ -27849,8 +27849,8 @@ ciqual_food_name:en:Gnocchi, from semolina, cooked
 ciqual_food_name:fr:Gnocchi à la semoule, cuit
 
 <en:Gnocchi of durum wheat semolina
-en:Raw gnocchi from semolina
-fr:Gnocchi à la semoule cru
+en:gnocchi from semolina
```

Comment:
"gnocchi from semolina" should just be a synonym of "Gnocchi of durum wheat semolina"

Reply:
fixed